README.md

```diff
@@ -47,7 +47,7 @@ You can install `inf-clojure` using the following command:
 
 or if you'd rather keep it in your dotfiles:
 
-```el
+```emacs-lisp
```

Comment:
Are you sure about this? I've been using `el` for years and I think it's just an alias for `cl`. I guess `emacs-lisp` could be the same.

Reply:
<img width="954" alt="image" src="https://user-images.githubusercontent.com/6377293/79461588-f983d580-7fbb-11ea-961a-16c05a4a6a25.png">
 Github seems to like it.

Reply:
https://github.com/github/linguist/blob/master/lib/linguist/languages.yml#L1342

Reply:
Good to know. Seems I've messed up something. :)

Reply:
Btw, I figure out why I was confused - turns out it's `el` in pygments (e.g. when you're using RTD or MkDocs). Too many doc systems. :D